src/components/ToastTextToCad.tsx

```diff
@@ -368,11 +392,9 @@ export function ToastTextToCadSuccess({
               onClick={() => {
                 // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 sendTelemetry(modelId, 'accepted', token)
-                navigate(
-                  `${PATHS.FILE}/${encodeURIComponent(
-                    `${context.project.path}${window.electron.sep}${data.fileName}`
-                  )}`
-                )
+                const path = `${projectDirectoryPath}${window.electron.path.sep}${projectName}${window.electron.sep}${fileName}`
+                navigate(`${PATHS.FILE}/${encodeURIComponent(path)}`)
```

Comment:
I think there's a gotcha here in that systemIOMachine's `requestedProjectName` and `requestedFileName` won't be updated because you're just navigating into them. But maybe that's not important?

Reply:
You could do the `nagivateToProject` thing with a `subRoute` if you wanted those context items to be updated on navigation